pkg/policy/selectorcache.go

```diff
@@ -129,6 +130,7 @@ type identitySelector interface {
 	addUser(CachedSelectionUser) (added bool)
 	removeUser(CachedSelectionUser) (last bool)
 	notifyUsers(added, deleted []identity.NumericIdentity)
+	numUsers() int
```

Comment:
IMO this is OK for now, but ee could also add a call to the Users interface to return a friendly string name for the user, then add a call here to return a slice of those names? Then we could show the list of users (selector policies).

Reply:
Sure, we can add these as time goes on. I think right now this is enough for debugging refcount issues and seeing whether stale identities are left behind, identities are mapped correctly, etc.